Makefile

```diff
@@ -604,3 +604,49 @@ install-wwhrd:
 
 check-licenses: install-wwhrd
 	wwhrd check -f .wwhrd.yml
+
+tsbs: tsbs-build tsbs-generate-data tsbs-load-data tsbs-generate-queries tsbs-run-queries
```

Comment:
this command should have a comment explaining roughly what it does and that it expects VM up&running at :8428 port. It should also mention how to change the path for ingestion and querying if, for example, user wants to run against cluster.

It should also mention original docs https://github.com/timescale/tsbs/blob/master/docs/victoriametrics.md